Src/Plugins/Security/SEC101_004.FacebookAppCredentialsValidator.cs

```diff
@@ -126,13 +126,13 @@ private ValidationState RetrieveInformation<T>(string url, string id, ref string
 
             try
             {
-                using HttpResponseMessage httpResponse = httpClient.GetAsync(url).GetAwaiter().GetResult();
+                using HttpResponseMessage response = httpClient.GetAsync(url).GetAwaiter().GetResult();
```

Comment:
>response [](http://example.com/codeflow?start=42&length=8)

let's continue to converge on consistent patterns for variable renaming etc. Also, very important: we need to rework our HTTP client creation/usage. The current model is not correct, per information provided by @cfaucon recently. I'll open an issue.

Reply:
Created the PR: [https://github.com/microsoft/sarif-pattern-matcher/pull/451](https://github.com/microsoft/sarif-pattern-matcher/pull/451)